### PR TITLE
Added Address and Company [fa_IR]

### PR DESCRIPTION
**Pull request by thisissorna**

### CODE REVIEW

**Comment by fzaninotto** [comment]
Please don't copy methods from the class you inherit.

**Comment by thisissorna** [comment]
hi
sorry its for speed

**Comment by fzaninotto** [comment]
since you add new formatters, please mention them in the locale-specific section of the README.

**Comment by fzaninotto** [comment]
Very bad rebase. Please include only your commits.

**Comment by thisissorna** [comment]
is there any problem?

File: readme.md [hunk 1]
```diff
@@ -940,6 +940,32 @@ echo $faker->vat;           // "A35864370"
 echo $faker->dni; // '83367512'
 ```
 
+### `Faker\Provider\fa_IR\Address`
+
+```php
+<?php
+// Generates a random address
+echo $faker->address; // "استان زنجان خیابان بهار ساختمان آفتاب پلاک 24"
```
Comment (fzaninotto) [inline readme.md]: this is not a local specific formatter please remove from readme

File: readme.md [hunk 2]
```diff
@@ -940,6 +940,32 @@ echo $faker->vat;           // "A35864370"
...
+echo $faker->city // "استان زنجان"
+```
+
+### `Faker\Provider\fa_IR\Company`
+
+```php
+<?php
+
+// Generates a random company name
+echo $faker->company; // "شرکت نساجی امیدی"
```
Comment (fzaninotto) [inline readme.md]: this is not a local specific formatter please remove from readme

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**